README.md

```diff
@@ -326,6 +326,13 @@ See [Pooling in the Microsoft Data Access
 Components](https://msdn.microsoft.com/en-us/library/ms810829.aspx#Troubleshooting%20MDAC%20Pooling) for more
 information.
 
+## Encoding
+
+If the file is in a different encoding use the CodePageIdentifer so the setting is passed to the engine.
+
+See [Code Page Identifiers](https://docs.microsoft.com/en-us/windows/desktop/intl/code-page-identifiers) for a complete 
+listing of Code Page Identifiers and their corresponding encoding.
+
```

Comment:
Please add a specific code example showing users how, and when, to set this new property.

Reply:
Done 👍